plugin/al/CMakeLists.txt

```diff
@@ -98,4 +98,4 @@ get_property(PYTHON_LIBRARY_LOCATION GLOBAL PROPERTY GLOBAL_PYTHON_LIBRARY_LOCAT
 configure_file(ALUsdMayaConfig.cmake.in ${PROJECT_BINARY_DIR}/ALUsdMayaConfig.cmake @ONLY)
 
 install(CODE "message(STATUS \"POST INSTALL: Compiling python/pyc for ${AL_INSTALL_PREFIX} ... \")")
-install(CODE "execute_process(COMMAND ${Python_EXECUTABLE} -m compileall ${AL_INSTALL_PREFIX} )")
+install(CODE "execute_process(COMMAND \"${Python_EXECUTABLE}\" -m compileall ${AL_INSTALL_PREFIX} )")
```

Comment:
wrap all the full path variables around quotes in case there are located in weird places with whitespaces and special characters on Windows.  Just to be safe!